documentation/docs/developers/strategies/tutorial.md

```diff
@@ -260,4 +260,4 @@ You can also run the `history` command to see the results of the trade:
 
 Congratulations - you have just created your first trading bot! This bot is very simple but should provide the foundation for you to experiment further. Can you prompt the user to change the order amount or trade type, or chain a series of trades?
 
-Before you know it, you will be creating complex trading strategies combining different exchanges with Hummingbot! To learn more about creating Hummingbot strategies, check out our [Developer Tutorial](/developers/contributions/).
\ No newline at end of file
+Before you know it, you will be creating complex trading strategies combining different exchanges with Hummingbot! To learn more about creating Hummingbot strategies, check out our [Developer Tutorial](/developers/contributions/).
```

Comment:
Some automatic Github linter removed the extra space at the end of this line, not directly related to my change.